cloudprem/physical/static/bastion_userdata.yml

```diff
@@ -28,7 +28,7 @@ write_files:
     #!/bin/bash -xe
     set -o xtrace
 
-    curl -L https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl \
+    curl -L curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.23.6/bin/linux/amd64/kubectl \
```

Comment:
`curl -L url -L0`? Is that an intentional double curl?

Reply:
Yes... uh... I was just checking to make sure you were paying attention.. 😂